Makefile

```diff
@@ -73,7 +73,13 @@ pylint:
 	       --msg-template='{C}: {path}:{line}:{column}: {msg} ({symbol})'
 
 # TODO: add test and code quality checks for `examples`
-# TODO: add test for IPython notebooks in gammapy-extra
+
+gammapy-extra: 
+	git clone https://github.com/gammapy/gammapy-extra.git
 
 doc-show:
 	open docs/_build/html/index.html
+
+test-notebooks: gammapy-extra
```

Comment:
No, I don't see a need to integrate notebook testing with `setup.py test`.
I think `make test-notebooks` is simpler and just fine.